geo/src/algorithm/bool_ops/assembly.rs

```diff
@@ -149,7 +150,7 @@ impl<T: GeoFloat> RegionAssembly<T> {
                     }
                     parent_snake_idx = parent.parent_snake_idx;
                 }
-                let this_children = children.entry(parent_ring_idx).or_insert(vec![]);
+                let this_children: &mut Vec<_> = children.entry(parent_ring_idx).or_default();
```

Comment:
Without the `or_insert` argument, type inference doesn't work, so we have to add a more explicit type annotation.